src/Symfony/Component/FeatureFlag/DataCollector/FeatureCheckerDataCollector.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\FeatureFlag\DataCollector;
+
+use Symfony\Component\FeatureFlag\FeatureRegistry;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\DataCollector\DataCollector;
+use Symfony\Component\HttpKernel\DataCollector\LateDataCollectorInterface;
+
+final class FeatureCheckerDataCollector extends DataCollector implements LateDataCollectorInterface
+{
+    private array $isEnabledLogs = [];
+    private array $valueLogs = [];
+
+    public function __construct(
+        private readonly FeatureRegistry $featureRegistry,
+    )
+    {
+        $this->data = [
+            'features' => [],
+        ];
+    }
+
+    public function collect(Request $request, Response $response, \Throwable $exception = null): void
+    {
+    }
+
+    public function collectIsEnabled(string $featureName, bool $result): void
```

Comment:
enableCollection ? 

Maybe personal, but at first it does not feel like a _setter_, more like a `isser` to me

Reply:
`enableCollection` looks like "I want to enable the collect of data". Since this method collects the result of `FeatureChecker::isEnabled`, it could be named `collectCheck`. WDYT ?

Reply:
Ok i understand it better now, that seems logical as you wrote it. You can forget this comment :)